src/startup.rs

```diff
@@ -0,0 +1,146 @@
+use arbiter_core::environment::{builder::EnvironmentBuilder, cheatcodes::Cheatcodes, Environment};
+use ethers::types::U256 as eU256;
+
+use super::*;
+
+// pub async fn initialize() -> Result<()> {
+//     let (mut _manager, admin, _alice, _bob, _multisig) = startup::set_up().await?;
+
+//     todo!()
+// }
+
+/// All the possible contracts that this simulation will actively use, but not
+/// all that are deployed!
+/// Each is bound to a `Client` and can be used to interact with the contract.
+/// The client in this case will be the admin.
+#[derive(Clone, Debug)]
+pub struct SimulationContracts {
+    /// The `L2OutputOracle` contract.
+    pub l2_output_oracle: L2OutputOracle<RevmMiddleware>,
+
+    /// The `BlockOracle` contract.
+    pub block_oracle: BlockOracle<RevmMiddleware>,
+
+    /// The `AlphabetVM` contract.
+    pub alphabet_vm: AlphabetVM<RevmMiddleware>,
+
+    /// The `FaultDisputeGame` contract.
+    pub disputegame: FaultDisputeGame<RevmMiddleware>,
+}
+
+pub async fn set_up_agents() -> Result<(
```

Comment:
Would it make it easier or more difficult if the return variable was a vec of agents?

Reply:
not sure, i think we need to have them ordered so we need to keep track of who is who.